### PR TITLE
feat!: Issue filters

### PR DESCRIPTION
**Pull request by drochow**

### CODE REVIEW

File: internal/api/graphql/graph/baseResolver/component.go [hunk 1]
```diff
@@ -106,3 +106,40 @@ func ComponentBaseResolver(app app.Heureka, ctx context.Context, filter *model.C
 	return &connection, nil
 
 }
+
+func ComponentNameBaseResolver(app app.Heureka, ctx context.Context, filter *model.ComponentFilter) (*model.FilterItem, error) {
+	requestedFields := GetPreloads(ctx)
+	logrus.WithFields(logrus.Fields{
+		"requestedFields": requestedFields,
+	}).Debug("Called IssueNameBaseResolver")
```
Comment (MR2011) [inline internal/api/graphql/graph/baseResolver/component.go:114]: Rename to ComponentNameBaseResolver

File: internal/database/mariadb/component.go [hunk 2]
```diff
@@ -267,3 +267,52 @@ func (s *SqlDatabase) DeleteComponent(id int64) error {
 
 	return err
 }
+
+func (s *SqlDatabase) GetComponentNames(filter *entity.ComponentFilter) ([]string, error) {
+	l := logrus.WithFields(logrus.Fields{
+		"filter": filter,
+		"event":  "database.GetServiceNames",
```
Comment (MR2011) [inline internal/database/mariadb/component.go:274]: rename to GetComponentNames

File: internal/database/mariadb/issue.go [hunk 3]
```diff
@@ -476,3 +476,52 @@ func (s *SqlDatabase) RemoveComponentVersionFromIssue(issueId int64, componentVe
 
 	return err
 }
+
+func (s *SqlDatabase) GetIssueNames(filter *entity.IssueFilter) ([]string, error) {
+	l := logrus.WithFields(logrus.Fields{
+		"filter": filter,
+		"event":  "database.GetServiceNames",
```
Comment (MR2011) [inline internal/database/mariadb/issue.go:483]: rename to GetIssueNames

**Review by MR2011 — APPROVED** [review]

**Comment by drochow** [comment]
For MVP delivery we did decide to omit tests for the new functionalities/tests and do them in a separate PR

**Review by MR2011 — APPROVED** [review]

**Pull request merged**